lib/CustomPropertiesConfig/CustomPropertiesConfigForm.js

```diff
@@ -25,21 +28,31 @@ class TermsConfigForm extends React.Component {
   sendCallout = (operation, outcome, error = '') => {
     this.callout.sendCallout({
       type: outcome,
-      message: <FormattedMessage id={`stripes-erm-components.terms.callout.${operation}.${outcome}`} values={{ error }} />,
+      message: <FormattedMessage id={`stripes-erm-components.${this.getSettingsRoute()}.callout.${operation}.${outcome}`} values={{ error }} />,
       timeout: error ? 0 : undefined, // Don't autohide callouts with a specified error message.
     });
   }
 
-  sendCalloutTermInUse = () => {
-    this.callout.sendCallout({
-      type: 'error',
-      message: <SafeHTMLMessage id="stripes-erm-components.terms.callout.delete.termInUse" />,
-      timeout: 0,
-    });
+  sendCalloutInUse = () => {
+    const translationKey = `stripes-erm-components.customProperties.callout.delete.${(this.getSettingsRoute() === 'terms') ? 'term' : 'supplementaryProperty'}InUse`;
+
+    return (
+      this.callout.sendCallout({
+        type: 'error',
+        message: <SafeHTMLMessage id={translationKey} />,
+        timeout: 0,
+      })
+    );
+  }
+
+  getSettingsRoute = () => {
```

Comment:
This feels pretty flaky. Can we just pass in `terms` or `customProperties` as a prop and use that instead?

Reply:
yeah, I am working on it locally and have made that change as I now moved the `CustomPropertiesConfigForm` to the respective apps (from where I pass in the `terms` or `supplementaryProperties`) and just have the `CustomPropertiesConfigListFieldArray` as a shared component